res/templates/pageVector2022.html

```diff
@@ -12,10 +12,7 @@
     <link rel="stylesheet" type="text/css" href="__RELATIVE_FILE_PATH__footer.css" />
     <link rel="stylesheet" type="text/css" href="__RELATIVE_FILE_PATH__vector-2022.css" />
   </head>
-  <body
-    class="skin--responsive skin-vector skin-vector-search-vue mediawiki ltr sitedir-ltr mw-hide-empty-elt ns-0 ns-subject mw-editable skin-vector-2022 action-view uls-dialog-sticky-hide"
-    cz-shortcut-listen="true"
-  >
+  <body class="__ARTICLE_BODY_CSS_CLASS__" cz-shortcut-listen="true">
```

Comment:
Why do we add some random `cz-shortcut-listen` to the body? It's not something set by the wiki and google suggests it's from the colorzilla browser extension.

Reply:
Because I copy-pasted it from my browser which has Colorzilla extension 🥺 

Will remove it.